configuration/pih/scripts/global/expandable.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Requires, given an id '[id]':
+ *   - Elements with IDs '[id]-[n]' where '[n]' counts the elements (starting from 1)
+ *   - A button with ID 'show-more-[id]'
+ *   - A button with ID 'show-less-[id]'
+ */
```

Comment:
I guess it risks become stale quickly, but perhaps add a comment here that links to an example of this in use?

Reply:
I added an example to the comment instead (I'd actually misread your comment as suggesting such the first time I read it). Does that seem good?

Reply:
Even better.